volume/drivers/fake/fake.go

```diff
@@ -811,3 +812,19 @@ func (d *driver) CloudBackupSchedEnumerate() (*api.CloudBackupSchedEnumerateResp
 		Schedules: schedules,
 	}, nil
 }
+
+func (d *driver) Catalog(volumeID, path, depth string) (api.CatalogResponse, error) {
+	return api.CatalogResponse{
+		Root: &api.Catalog{
+			Name:         "",
+			Path:         "/var/lib/osd/catalog/12345678",
+			Type:         "Directory",
+			Size:         4096,
+			LastModified: &timestamp.Timestamp{},
+		},
+		Report: &api.Report{
+			Directories: 0,
+			Files:       0,
+		},
+	}, volume.ErrNotSupported
```

Comment:
Ha!, you added support then returned not supported?

Reply:
whahah right :)